internal/services/mariadb/mariadb_server_resource.go

```diff
@@ -150,6 +150,13 @@ func resourceMariaDbServer() *pluginsdk.Resource {
 				Required: true,
 			},
 
+			"ssl_minimal_tls_version_enforced": {
+				Type:         pluginsdk.TypeString,
+				Optional:     true,
+				Default:      string(servers.MinimalTlsVersionEnumTLSOneTwo),
+				ValidateFunc: validation.StringInSlice(servers.PossibleValuesForMinimalTlsVersionEnum(), false),
+			},
```

Comment:
Whilst I agree from a security perspective that this should be defaulted to `TLS1_2`, it looks like the API is defaulting this to `TLSEnforcementDisabled`. Perhaps this should be `Computed` for now and default to `TLS1_2` in 4.x?